docs/install/install-edex.md

```diff
@@ -275,3 +274,114 @@ This file specifies configuration and runtime parameters. If you are pulling in
 ```
 rpm -qa | grep awips2-edex
 ```
+
+## Uninstalling EDEX
+These are instructions to manually uninstall EDEX. However, the [`awips_install.sh`](#1-install-edex) script will do all of these steps for you if you are installing a newer version of EDEX.
+
+**1. Make sure all EDEX processes are stopped**
+```
+sudo edex stop
+sudo edex status
+
+[edex status]
+ postgres    :: not running
+ pypies      :: not running
+ qpid        :: not running
+ EDEXingest  :: not running
+ EDEXgrib    :: not running
+ EDEXrequest :: not running
+ ldmadmin    :: not running
+```
+
+**2. Backup any important configuration files that you may want to reference**
+
+Here are some possible important directories/files to backup:
+
+```
+/awips2/database/data/pg_hba.conf
+/awips2/edex/data/utility
+/awips2/edex/bin
+/awips2/ldm
+/awips2/dev
+/awips2/edex/conf
+/awips2/edex/etc
+/awips2/edex/logs
+/usr/bin/edex
+/etc/init.d/edexServiceList
+```
+
+**3. See what AWIPS yum groups are currently installed**
+
+In this case the "AWIPS EDEX Server" group installed
```

Comment:
maybe...
"In this case, the "AWIPS EDEX Server" group **is** installed"?

Do you think putting AWIPS EDEX Server in code quotes would be appropriate? or just the normal quotes?